techniques/system/inventory/1.0/fusionAgent.st

```diff
@@ -115,6 +125,8 @@ bundle agent doInventory_always
       "any" usebundle => moveInventoryToFinalDestination;
       "any" usebundle => signInventories;
       "any" usebundle => compressInventories;
+
+    uuid_valid.(resend_inventory|force_inventory|((inventory_time|initial_promises).!inventory_sent)).inventory_valid::
```

Comment:
inventory_valid is defined by bundle checkInventoryFile
This condition can never be true if class resend_inventory is set, as it would not define the inventroy_valid class
I suggest you use:
(uuid_valid.(force_inventory|((inventory_time|initial_promises).!inventory_sent)).inventory_valid)|resend_inventory::
so if resend_inventory is set, it would send the inventory always

What do you think ?

Reply:
Good idea